terracottabigmemorymax/helm/values.yaml

```diff
@@ -76,6 +76,8 @@ terracotta:
   tmcOpts: ""
   # -- The license content for the Terracotta cluster. Optional.
   license: ""
+  # -- Can be used for enabling prometheus operator service monitor for scraping metrics.
+  serviceMonitorEnabled: false
```

Comment:
Is it duplicated? see `serviceMonitor` in line 153

Reply:
need to remove this